api/model/src/main/java/org/projectnessie/model/MergeKeyBehavior.java

```diff
@@ -28,6 +33,24 @@ public interface MergeKeyBehavior {
 
   MergeBehavior getMergeBehavior();
 
+  @JsonInclude(Include.NON_NULL)
+  @JsonView(Views.V2.class)
+  @Nullable
+  @jakarta.annotation.Nullable
+  Content getExpectedSourceContent();
+
+  @JsonInclude(Include.NON_NULL)
+  @JsonView(Views.V2.class)
+  @Nullable
+  @jakarta.annotation.Nullable
+  Content getExpectedTargetContent();
```

Comment:
Could you add javadoc for how this parameter is supposed to work? I assume if it is set, but does not match the target HEAD's content, that will be a conflict, right?

What if it does not match the target's "expected hash" content? Will that be a "bad request"?

Reply:
Yes, should result in a conflict.